libgalois/include/katana/DynamicBitset.h

```diff
@@ -92,6 +92,19 @@ class KATANA_EXPORT DynamicBitset {
     bitvec.reserve((n + bits_uint64 - 1) / bits_uint64);
   }
 
+  /**
+   * Clears the bitset.
+   */
+  void clear() {
```

Comment:
Seems like this file is a mix of LLVM and Google style function names. Can we be consistent at least with new code?

Reply:
Can you please clarify what you want changed? I'm using the same `clang-format` on it as before.

Reply:
Sorry, `Clear` versus `clear`.

Reply:
I want it to be consistent with `std::vector`. That's why I used `clear()` and `shrink_to_fit()`.

Reply:
ok `clear` seems fine then, but is there a vector operation called `shrink_to_fit()`? And why do we have `GetOffsetsInto`?

Reply:
Yes, there is `shrink_to_fit`. I didn't know until I saw Simon's changes recently. 

`GetOffsetsInto` (now `AppendOffsets`) will be used in the partitioner. I want to avoid a copy.

Reply:
Oh. I didn't realize that `shrink_to_fit` was a vector function already.

You can ignore this comment then.